src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -476,6 +476,14 @@ export class NotebookModel extends Disposable implements INotebookModel {
 		}
 	}
 
+	public createNotebookTelemetryEvent(action: TelemetryKeys.TelemetryAction | TelemetryKeys.NbTelemetryAction, additionalProperties?: ITelemetryEventProperties): void {
```

Comment:
`sendNotebookTelemetryEvent`

Also please update other places we're sending Notebook events to use this as well (there's more outside of just the model class)

Reply:
there are some classes that do not use NotebookModel (NotebookExplorerViewlet). 
For now, I'll update the ones that have a notebook model instance and I can address the remaining places in a different pr.

Reply:
Yeah, those ones probably wouldn't be associated with a specific Notebook anyways so that's fine. This helper is specifically for telemetry around Notebooks themselves. But worth taking a look as a follow up.